README.md

```diff
@@ -51,6 +54,22 @@ This repository contains build files for the FreeRTOS Windows port - able to be
 
 In both cases, the generated executable can be found at `build/Cortex-M3_MPS2_QEMU_GCC/output/RTOSDemo.elf`.
 
+**Running the makefile build in QEMU**
+
+Use the QEMU command line contained in the [start_qemu.bat](https://github.com/FreeRTOS/coreMQTT-Agent-Demos/blob/main/build/Cortex-M3_MPS2_QEMU_GCC/start_qemu.bat) batch file to run the generated 'RTOSDemo.elf' file directly.  Alternatively use the command line contained in the [start_qemu_and_wait.bat](https://github.com/FreeRTOS/coreMQTT-Agent-Demos/blob/main/build/Cortex-M3_MPS2_QEMU_GCC/start_qemu_and_wait.bat) batch file to start QEMU and wait for a GBD connection on port 1234.  The Eclipse project contains a suitable debug configuration.
```

Comment:
We should refer to it with a relative link since it's in the same repository. Also "QEMU command line" - can this just be "QEMU command"?
```suggestion
Use the QEMU command contained in the [start_qemu.bat](build/Cortex-M3_MPS2_QEMU_GCC/start_qemu.bat) batch file to run the generated `RTOSDemo.elf` file directly.  Alternatively, use the command contained in the [start_qemu_and_wait.bat](build/Cortex-M3_MPS2_QEMU_GCC/start_qemu_and_wait.bat) batch file to start QEMU and wait for a GBD connection on port 1234.  The Eclipse project contains a suitable debug configuration.
```

Reply:
Should this be GBD or GDB? Also, what is the Eclipse debug configuration suitable for? Should it instead be "configuration suitable for debugging"?
```suggestion
Use the QEMU command contained in the [start_qemu.bat](build/Cortex-M3_MPS2_QEMU_GCC/start_qemu.bat) batch file to run the generated `RTOSDemo.elf` file directly.  Alternatively, use the command contained in the [start_qemu_and_wait.bat](build/Cortex-M3_MPS2_QEMU_GCC/start_qemu_and_wait.bat) batch file to start QEMU and wait for a GDB connection on port 1234.  The Eclipse project contains a configuration suitable for debugging.
```